maestro-client/build.gradle

```diff
@@ -65,6 +65,13 @@ dependencies {
     api 'org.mozilla:rhino:1.7.14'
     api 'org.jsoup:jsoup:1.15.3'
     api 'com.squareup.okhttp3:okhttp:4.10.0'
+    api 'org.rauschig:jarchivelib:1.2.0'
+    api 'com.fasterxml.jackson.core:jackson-databind:2.13.4'
+    api 'com.fasterxml.jackson.module:jackson-module-kotlin:2.13.4'
+    api 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.13.4'
+    api 'com.fasterxml.jackson.dataformat:jackson-dataformat-xml:2.13.4'
+    api 'commons-io:commons-io:2.11.0'
```

Comment:
What do we need commons-io for? Kotlin typically has many of the common IO utils, so just double checking whether we really need it

Reply:
Removed this and used File.deleteRecursively